### DISCUSSION TITLE
Some general questions about the Data importer

### DISCUSSION THREAD

**Discussion by mh4ckt3mh4ckt1c4s** [firefly-iii/firefly-iii#6683] [General]
Hello everyone, 

I started to use Firefly and the FIDI and I was wondering : 
- Why is the data importer separated from Firefly ? Is there a specific reason I was not able to find ?
- Is it planned to merge FIDI into Firefly ?

**Comment by JC5** [comment]
https://docs.firefly-iii.org/data-importer/help/faq/#why-isnt-the-data-importer-built-into-firefly-iii

The data importer used to be a part of Firefly III but I separated them on purpose.

**Comment by mh4ckt3mh4ckt1c4s** [comment]
Thanks, I read all the Firefly FAQ but forgot about the FIDI one. I'm using Firefly on Yunohost and I'd like to see FIDI arrive on it soon. I don't know if it is still in development, or what is the meaning of the 0.9.x releases ? Is FIDI suitable to be deployed on production yet or are there some fixes / features to wait for ?

**Reply by JC5** [reply]
It is suitable to be deployed already, but I'm not actively working on a Yunohost release myself.

**Comment by mh4ckt3mh4ckt1c4s** [comment]
Okay, thanks for the clarifications !
